test/apiv2/20-containers.at

```diff
@@ -85,7 +85,7 @@ else
     fi
 fi
 
-t DELETE libpod/containers/$cid 204
+t DELETE libpod/containers/$cid 200
```

Comment:
All these 204->200 changes trouble me. @jwhonce could you please confirm whether or not these are OK?

Reply:
This is the breaking API.  200 means no data is returned 204 Means data is returned, which is  new, As I understand it.

Reply:
Other way around: 200 returns data, 204 does not. 

One other thing, this change should probably be reflected in the swagger docs. I don't see that as part of this PR, but it's been too long since I looked at swagger so I may just be missing something obvious.

Reply:
Oh and one last thing: since 200 returns data, if this is really truly a desired change, could you add some data checks to these tests? Something like
```diff
-t DELETE libpod/containers/$cid 200
+t DELETE libpod/containers/$cid 200 length=1 .[0].Id=$cid
```

Reply:
Ok I updated Swagger, Yes I screwed up.  Before if you removed a container you got back no data, now you get back a list of containers that were removed or failed to be removed.

Basically if you send in the depend flag, the rm command can remove one or more containers.

Eventually we should move options.All to the server side as well.